pkg/schedule/schedulers/scheduler_controller.go

```diff
@@ -456,6 +456,7 @@ func (s *ScheduleController) Stop() {
 
 // Schedule tries to create some operators.
 func (s *ScheduleController) Schedule(diagnosable bool) []*operator.Operator {
+loop:
```

Comment:
Can we put `cacheCluster := newCacheCluster(s.cluster)` at the very beginning of the loop?